script/Arcueid/Day 11/11_00_ARC11_2.txt

```diff
@@ -765,7 +766,7 @@ So then, why?
 [sha:bd822e773e2a3479845a95973f7b4076c658069c]{
 -- Page 87, Offset 29749.
 -- あいつは自分自身がもうダメだって判っていたのに、こんなところで吸血鬼退治なんかをやっていたのか。
-If she herself knew that she couldn't go on anymore, then what was she doing hunting vampires in a place like this?
+If she herself knew that she couldn't go on anymore, then what was she doing, hunting vampires in a place like this?
```

Comment:
Prefer original

Reply:
agreed. i think they have subtly different meanings and the original is correct